BibBuddy/app/src/main/java/de/bibbuddy/MainActivity.java

```diff
@@ -23,6 +23,7 @@
   BottomNavigationView bottomNavigationView;
   FragmentManager fragmentManager;
   DatabaseHelper dbHelper;
+  private ImageButton logoButton;
```

Comment:
Please add a newline for better readability.
```suggestion

  private ImageButton logoButton;
```

Reply:
Thanks for the suggestion, done!